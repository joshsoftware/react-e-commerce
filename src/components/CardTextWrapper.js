import React from 'react';
import { CardText } from 'reactstrap';
import { PropTypes } from 'prop-types';

const CardTextWrapper = ({ text, className }) => {
  return <CardText className={className}>{text}</CardText>;
};

export default CardTextWrapper;

CardTextWrapper.propTypes = {
  text: PropTypes.string.isRequired,
  className: PropTypes.string
};
