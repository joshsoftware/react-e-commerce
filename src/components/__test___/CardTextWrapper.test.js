import React from 'react';
import { render } from '@testing-library/react';
import CardTextWrapper from '../CardTextWrapper';

describe('Card Text', () => {
  it('Must Return Card Text', () => {
    const { asFragment } = render(<CardTextWrapper text={'Card Text'} />);
    expect(asFragment()).toMatchSnapshot();
  });
});
