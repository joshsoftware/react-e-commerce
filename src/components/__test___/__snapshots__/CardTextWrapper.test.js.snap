// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card Text Must Return Card Text 1`] = `
<DocumentFragment>
  <p
    class="card-text"
  >
    Card Text
  </p>
</DocumentFragment>
`;
